### ISSUE TITLE
Support for thermal_zone temperatures by name

### ISSUE THREAD

**Issue by jimis** [vmatare/thinkfan#43]
Feature request:

I'm using `thermal_zone` temperatures instead of `hwmon` because the paths are more predictable and do not disappear (unlike the `temp[0-9]_input` hwmon files). Here are the paths I'm using:
```
$ cat /sys/devices/virtual/thermal/thermal_zone[0-3]/temp
43000
40500
30000
44000
```

However they might still change order, so I would like to set each sensor by the name instead of the path. Names are stored in the `type` files:
```
$ cat /sys/devices/virtual/thermal/thermal_zone[0-3]/type
acpitz
pch_skylake
iwlwifi
x86_pkg_temp
```

This way I would adjust the parameters more accurately, for example if I want to exclude a specific sensor (for example the `iwlwifi` that disappears if wifi is disabled).

**Comment by phrxmd** [comment]
This would be pretty neat, also because the `thermal_zone` tree usually provides much more fine-grained control over temperature data.

From an UI side It could be made accessible through an extra keyword for temperature inputs in `thinkfan.conf`, in addition to the existing `tp_thermal`, `hwmon` and `atasmart` keywords. E.g. you'd have an extra line:

```
thermal_zone pch_skylake
```

that would give you the temperature input from whatever `thermal_zone*` subtree has  `pch_skylake` in its `type` file.

**Comment by akheron** [comment]
I created a PR to fix this: #69
